jwst/outlier_detection/tests/test_outlier_detection.py

```diff
@@ -153,11 +173,91 @@ def test_outlier_step_no_outliers(we_three_sci):
 
 
 def test_outlier_step(we_three_sci):
-    """Test whole step with an outlier"""
+    """Test whole step with an outlier including saving intermediate and results files"""
```

Comment:
Please use the `_jail` fixture here so that intermediate files are written to  a tmpdir.  I.e.:

```python
def test_outlier_step(_jail, we_three_sci):
    ...
```

Reply:
Will do.  Never used this before, so hopefully just doing as you say is all I need to do.  If there was a pointer to docs on what this does, that would be awesome.

Reply:
You think we have time to write documentation for developers?!  😆 

It is documented in the package we get it from

https://ci-watson.readthedocs.io/en/latest/#plugin

but there's no documentation in `jwst` that we actually use this plugin.  😅

Reply:
Neat.  Thanks.